src/components/workspace-utils.js

```diff
@@ -59,7 +59,7 @@ export const WorkspaceSelector = ({ workspaces, value, onChange, ...props }) =>
 export const WorkspaceImporter = _.flow(
   Utils.withDisplayName('WorkspaceImporter'),
   withWorkspaces
-)(({ workspaces, refreshWorkspaces, onImport, authorizationDomain: ad, selectedWorkspaceId: initialWs }) => {
+)(({ workspaces, refreshWorkspaces, onImport, authorizationDomain: ad, selectedWorkspaceId: initialWs, isWorkflowInvalid = false }) => {
```

Comment:
Nit: I prefer boolean names based on the affirmative, i.e. `isWorkflowNameValid`. I've read this advice before but I don't remember where. I think it tends to avoid having to think through double negatives.